packages/material-ui/src/Paper/Paper.js

```diff
@@ -45,6 +46,17 @@ const PaperRoot = experimentalStyled(
     overridesResolver,
   },
 )(({ theme, styleProps }) => {
+  // Inspired by https://github.com/material-components/material-components-ios/blob/bca36107405594d5b7b16265a5b0ed698f85a5ee/components/Elevation/src/UIColor%2BMaterialElevation.m#L61
+  const getOverlayAlpha = (elevation) => {
```

Comment:
The function can be moved to the parent scope. No need to recreate it at each render.